google/cloud/spanner/.repo-metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+    "api_id": "spanner.googleapis.com",
+    "api_short_name": "spanner",
+    "client_documentation": "https://cloud.google.com/cpp/reference/spanner/latest",
+    "distribution_name": "google-cloud-cpp",
+    "issue_tracker": "https://issuetracker.google.com/issues?q=componentid:190851%2B%20status:open",
+    "language": "cpp",
+    "library_type": "GAPIC_COMBO",
+    "name_pretty": "Cloud Spanner Admin API",
```

Comment:
Why "Admin"?

Reply:
Thanks.  Fixed. FTR, I used a lot of scripts and I probably pulled this from a README or some generic YAML file.